pyanaconda/modules/users/users.py

```diff
@@ -36,8 +36,9 @@
 from pyanaconda.anaconda_loggers import get_module_logger
 log = get_module_logger(__name__)
 
-class UsersModule(KickstartModule):
-    """The Users module."""
+
+class UsersService(KickstartService):
+    """The Users services."""
```

Comment:
Shouldn't this the "The Users service" ? We have in singular everywhere else as far as I can tell.

Reply:
Fixed.